src/openms/include/OpenMS/FILTERING/BASELINE/MorphologicalFilter.h

```diff
@@ -172,10 +172,10 @@ namespace OpenMS
       //structuring element
       defaults_.setValue("struc_elem_length", 3.0, "Length of the structuring element. This should be wider than the expected peak width.");
       defaults_.setValue("struc_elem_unit", "Thomson", "The unit of the 'struct_elem_length'.");
-      defaults_.setValidStrings("struc_elem_unit", ListUtils::create<String>("Thomson,DataPoints"));
+      defaults_.setValidStrings("struc_elem_unit", ListUtils::create<std::string>("Thomson,DataPoints"));
```

Comment:
does replace by 
, {"Thomson", "DataPoints"})
work? would prefer - also in other files

Reply:
yes that would work, that may also remove the need for the ListUtils include in some files